docs/source/algorithms/ReorientUnitCell-v1.rst

```diff
@@ -0,0 +1,107 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+ReorientUnitCell finds the unit cell most aligned with the goniometer axes by finding rotation :math:`U`
+most similar to the identity matrix. Rotation :math:`U` transforms the reciprocal orthonormal
```

Comment:
We decided it's not true. U is the one where a* is mostly along the beam and b* is most in the horizontal plane